CHANGELOG.md

```diff
@@ -81,7 +85,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 * Per-repository configuration is now read from `.jj/repo/config.toml`.
 
-* The `ui.relative-timestamps` option now also affects `jj op log`.
+* The `ui.relative-timestamps` option now affects `jj op log`.
```

Comment:
Not necessarily for this CL: I think this option should now default to `true`, since that's much more useful fro the oplog.

If it is going to last for any length of time, it should be renamed to `ui.oplog-relative-timestamps`

Reply:
I won't include that in this PR, but `ui.oplog-relative-timestamps = true` makes sense.